engine/storage/snapshot/src/org/apache/cloudstack/storage/snapshot/SnapshotObject.java

```diff
@@ -61,7 +61,7 @@
     private static final Logger s_logger = Logger.getLogger(SnapshotObject.class);
     private SnapshotVO snapshot;
     private DataStore store;
-    private Object payload;
+    private Object payload, fullBackup;
```

Comment:
fullBackup stores a boolean value, if a snapshot is full or delta snapshot, payloads' value would be an object of CreateSnapshotPayload.